crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1289,10 +1278,13 @@ impl RpcServerConfig {
     /// Builds the ws and http server(s).
     ///
     /// If both are on the same port, they are combined into one server.
-    async fn build_ws_http(
+    pub async fn start_ws_http(
```

Comment:
this is now the new `start` function?

then we should rename this to start, and perhaps we should consume the value now?

Reply:
what do you mean by consume the value now